integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AwsCheckSpotInstance.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.it.cloudbreak;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.ec2.AmazonEC2Client;
+import com.amazonaws.services.ec2.model.DescribeSpotInstanceRequestsResult;
+import com.amazonaws.services.ec2.model.SpotInstanceRequest;
+import com.sequenceiq.cloudbreak.api.endpoint.StackEndpoint;
+import com.sequenceiq.cloudbreak.api.model.InstanceGroupResponse;
+import com.sequenceiq.cloudbreak.api.model.InstanceMetaDataJson;
+import com.sequenceiq.cloudbreak.api.model.StackResponse;
+import com.sequenceiq.it.IntegrationTestContext;
+
+
+public class AwsCheckSpotInstance extends AbstractCloudbreakIntegrationTest {
+
+    @Parameters({"region", "hostGroupsToCheck"})
+    @Test
+    public void checkSpotInstance(Regions region, String hostGroupsToCheck) {
+        //GIVEN
+        IntegrationTestContext itContext = getItContext();
+        String stackId = itContext.getContextParam(CloudbreakITContextConstants.STACK_ID);
+
+        StackEndpoint stackEndpoint = getCloudbreakClient().stackEndpoint();
+        StackResponse stackResponse = stackEndpoint.get(Long.valueOf(stackId));
+
+        List<InstanceGroupResponse> instanceGroups = stackResponse.getInstanceGroups();
+
+        List<String> instanceIdList = new ArrayList<>();
+        List<String> hostGroupList = Arrays.asList(hostGroupsToCheck.split(","));
+
+        for (InstanceGroupResponse instanceGroup : instanceGroups) {
+            if (hostGroupList.contains(instanceGroup.getGroup().toString())) {
+                Set<InstanceMetaDataJson> instanceMetaData = instanceGroup.getMetadata();
+                for (InstanceMetaDataJson metaData : instanceMetaData) {
+                    instanceIdList.add(metaData.getInstanceId());
+                }
+            }
+        }
+        //WHEN
+        AmazonEC2Client ec2 = new AmazonEC2Client();
+        ec2.setRegion(Region.getRegion(region));
+        DescribeSpotInstanceRequestsResult describeSpotInstanceRequestsResult = ec2.describeSpotInstanceRequests();
```

Comment:
I recommend to use the parametrized version of this function with the instance ids:
http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ec2/AmazonEC2Client.html#describeSpotInstanceRequests-com.amazonaws.services.ec2.model.DescribeSpotInstanceRequestsRequest-